locales/zh-TW.json

```diff
@@ -138,33 +154,69 @@
   "conversation": {
     "with": "與"
   },
+  "custom_cards": {
+    "stackblitz": {
+      "lines": "第 {0} 行",
+      "open": "開啟",
+      "snippet_from": "取自 {0} 的片段"
+    }
+  },
   "error": {
     "account_not_found": "未找到使用者 {0}",
     "explore-list-empty": "目前沒有熱門話題，稍後再來看看吧！",
-    "file_size_cannot_exceed_n_mb": "檔案大小不能超過 {0}MB",
+    "file_size_cannot_exceed_n_mb": "檔案大小不能超過 {0} MB",
     "sign_in_error": "無法連接伺服器",
     "status_not_found": "未找到貼文",
     "unsupported_file_format": "不支援的檔案格式"
   },
   "help": {
+    "build_preview": {
+      "desc1": "您當前正在查看來自開源社區、預覽版的鹿鳴 - {0}。",
+      "desc2": "可能包含末經審查或是惡意修改的內容。",
+      "desc3": "請不要使用真實的帳號登入。",
+      "title": "預覽部署"
+    },
     "desc_highlight": "可能會在某些地方出現一些 bug 或缺少的功能。",
     "desc_para1": "感謝你有興趣嘗試鹿鳴，一個我們正在積極開發的通用 Mastodon 用戶端。",
     "desc_para2": "我們正在努力開發中，並隨著時間的推移不斷完善。",
     "desc_para3": "為了幫助促進開發，你可以通過以下連結贊助我們的團隊成員。希望你喜歡鹿鳴！",
     "desc_para4": "鹿鳴是開源的，如果你願意幫助測試、提供回饋或作出貢獻，",
     "desc_para5": "在 GitHub 上聯繫我們",
     "desc_para6": "來參與其中。",
+    "footer_team": "鹿鳴開發團隊",
     "title": "預覽鹿鳴！"
   },
   "language": {
     "search": "搜尋"
   },
+  "list": {
+    "add_account": "在列表中添加帳號",
+    "cancel_edit": "取消編輯",
+    "clear_error": "清除錯誤",
+    "create": "建立",
+    "delete": "刪除這個列表",
+    "delete_error": "刪除列表時發生錯誤",
+    "edit": "編輯這個列表",
+    "edit_error": "更新列表時發生錯誤",
+    "error": "建立列表時發生錯誤",
+    "error_prefix": "錯誤：",
+    "list_title_placeholder": "列表標題",
+    "modify_account": "修改列表中的帳號",
+    "remove_account": "移除列表中的帳號",
+    "save": "保存變更"
+  },
   "menu": {
     "block_account": "封鎖 {0}",
     "block_domain": "封鎖的域名 {0}",
     "copy_link_to_post": "複製這篇貼文的連結",
+    "copy_original_link_to_post": "複製貼文的原網址",
     "delete": "刪除",
     "delete_and_redraft": "刪除並重新編輯",
+    "delete_confirm": {
+      "cancel": "取消",
+      "confirm": "確認",
+      "title": "你確定要刪除這篇貼文嗎？"
+    },
```

Comment:
This doesn't seem to be used in Elk.

Reply:
Then, should I remove these XD

Reply:
I think it's better to remove it.

Reply:
Of course, then I remove your mentioned parts.